src/CAN/can_mapping.c

```diff
@@ -32,6 +32,8 @@ float canmapping_extract_value(uint64_t raw_data, const CANMapping *mapping)
         }
         uint32_t bitmask = (1UL << length) - 1;
         uint32_t raw_value = (raw_data >> offset) & bitmask;
+
+        /* normalize endian */
```

Comment:
Thanks. if we do it now, I'd rather put it in capabilities.h as a #define . A convention also used in freeRTOS and ChibiOS and I don't think it's a bad approach.